app/models/spree/easypost/address_decorator.rb

```diff
@@ -0,0 +1,70 @@
+module Spree
+  module EasyPost
+    module AddressDecorator
+      def self.prepended(base)
+        base.validate :easypost_address_validate
```

Comment:
I would do easypost_address_validate if easypost_validated_address? and add easypost_validate_address to configs